conda/recipes/strings_udf/meta.yaml

```diff
@@ -1,78 +0,0 @@
-# Copyright (c) 2022-2023, NVIDIA CORPORATION.
-
-{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
-{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
-{% set py_version = environ['CONDA_PY'] %}
-{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
-{% set cuda_major = cuda_version.split('.')[0] %}
-{% set date_string = environ['RAPIDS_DATE_STRING'] %}
-
-package:
-  name: strings_udf
-  version: {{ version }}
-
-source:
-  git_url: ../../..
-
-build:
-  number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-  script_env:
-    - PARALLEL_LEVEL
-    - CMAKE_GENERATOR
-    - CMAKE_C_COMPILER_LAUNCHER
-    - CMAKE_CXX_COMPILER_LAUNCHER
-    - CMAKE_CUDA_COMPILER_LAUNCHER
-    - SCCACHE_S3_KEY_PREFIX=strings-udf-aarch64 # [aarch64]
-    - SCCACHE_S3_KEY_PREFIX=strings-udf-linux64 # [linux64]
-    - SCCACHE_BUCKET
-    - SCCACHE_REGION
-    - SCCACHE_IDLE_TIMEOUT
-    - AWS_ACCESS_KEY_ID
-    - AWS_SECRET_ACCESS_KEY
-  ignore_run_exports:
-    # libcudf's run_exports pinning is looser than we would like
-    - libcudf
-  ignore_run_exports_from:
-    - {{ compiler('cuda') }}
-
-requirements:
-  build:
-    - cmake {{ cmake_version }}
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
-    - ninja
-    - sysroot_{{ target_platform }} {{ sysroot_version }}
-  host:
-    - python
-    - cython >=0.29,<0.30
-    - scikit-build >=0.13.1
-    - setuptools
-    - numba >=0.54
-    - libcudf ={{ version }}
-    - cudf ={{ version }}
-    - cudatoolkit ={{ cuda_version }}
-  run:
-    - python
-    - typing_extensions
-    - numba >=0.54
-    - numpy
-    - libcudf ={{ version }}
-    - cudf ={{ version }}
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
-    - cachetools
-    - ptxcompiler >=0.7.0  # CUDA enhanced compatibility. See https://github.com/rapidsai/ptxcompiler
```

Comment:
We don't require `ptxcompiler >=0.7.0` in the `cudf` recipe. We should, right?

Reply:
Added this. Currently this is being inherited through `cubinlinker`.